.circleci/config.yml

```diff
@@ -4,16 +4,13 @@
 version: 2
 
 references:
-  workspace: &workspace
-               ~/work
```

Comment:
Personally I find these references more confusing than helpful for smaller pieces of config like this. I considered getting rid of `android_config` as well, but that is used a bunch of different places.